setup.py

```diff
@@ -40,10 +39,20 @@
 #                                                                              #
 ################################################################################
 
+import os
 import textwrap
 
 import setuptools
 
+_PATH_ROOT = os.path.dirname(__file__)
+_PATH_REQUIRES = os.path.join(_PATH_ROOT, "requirements")
+
+
+def _load_requirements(path_dir: str = _PATH_ROOT, file_name: str = "requirements.txt") -> "list[str]":
+    with open(os.path.join(path_dir, file_name)) as fo:
+        return [r.strip() for r in fo.readlines() if r.strip() and not r.strip().startswith("#")]
```

Comment:
I think this makes the list comprehension more readable (while `strip()` is being called only once):
```suggestion
def _load_requirements(path_dir: str = _PATH_ROOT, file_name: str = "requirements.txt") -> "list[str]":
    with open(os.path.join(path_dir, file_name)) as fo:
        return [requirement
                for line in fo.readlines()
                for requirement in [line.strip()]
                if requirement and not requirement.startswith("#")]
```

Reply:
maybe `[r for r in (x.strip() for x in f.readlines()) if r and not r.startswith("#")]` for more readable

Reply:
```suggestion
def _load_requirements(path_dir: str = _PATH_ROOT, file_name: str = "requirements.txt") -> "list[str]":
    with open(os.path.join(path_dir, file_name)) as fo:
        reqs = [ln.strip() for ln in fo.readlines()]
    return [r for r in reqs if r and not r.startswith("#")]
```